astropy/constants/iers2010.py

```diff
@@ -0,0 +1,103 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""
+Astronomical and physics constants in SI units.  See :mod:`astropy.constants`
+for a complete listing of constants defined in Astropy.
+"""
+
+import numpy as np
+
+from .constant import Constant
+
+
+class IERS2010(Constant):
+    default_reference = 'IERS 2010'
+    _registry = {}
+    _has_incompatible_units = set()
+
+
+# NATURAL DEFINING CONSTANTS
+c = IERS2010('c', "Speed of light in vacuum", 299792458.,
+             'm / (s)', 0.0, system='si')
+
+# AUXILIARY DEFINING CONSTANTS
+k = IERS2010('k', "Gaussian gravitational constant", 1.720209895e-2,
+             '', 0.0, system='si')
+
+L_G = IERS2010('L_G', "1−d(TT)/d(TCG)", 6.969290134e-10,
+               '', 0.0, system='si')
+
+L_B = IERS2010('L_B', "1−d(TDB)/d(TCB)", 1.550519768e-8,
+               '', 0.0, system='si')
+
+TDB0 = IERS2010('TDB0', "TDB-TCB at JD 2443144.5 TAI", -6.55e-5,
+                's', 0.0, system='si')
+
+theta0 = IERS2010('theta0', "Earth Rotation Angle (ERA) at J2000.0",
+                  0.7790572732640 * 2 * np.pi, 'rad', 0.0, system='si')
+
+dtheta = IERS2010('dtheta', "Rate of advance of ERA",
+                  1.00273781191135448 * 2 * np.pi, 'rad', 0.0, system='si')
```

Comment:
Same, but unit should be `cycle / day`. I would add to the docstring that it is 'revolution / UT1 day'